esm/interface/document.js

```diff
@@ -269,10 +269,10 @@ export class Document extends NonElementParentNode {
   createAttributeNS(_, name) {
     return this.createAttribute(name);
   }
-  createElementNS(nsp, localName, options) {
+  createElementNS(nsp, localName) {
     return nsp === SVG_NAMESPACE ?
             new SVGElement(this, localName, null) :
-            this.createElement(localName, options);
```

Comment:
options is used for builtin extends ... I think this breaks builtin extend when created via `createElementNS` but I need to check if browsers also do that

Reply:
checked here: I am afraid this is not going to lad as is https://github.com/WebReflection/linkedom/pull/190#discussion_r1103862030